docs/modules/ROOT/pages/how-to-guides/managing-environments/con_overview_of_environments.adoc

```diff
@@ -0,0 +1,27 @@
+= Overview of {ProductName} environments
+
+Environments represent infrastructure. You can use them to test, develop, and release your application to production. You can create two types of environments in {ProductName}:
+
+* Static environments, to which you can deploy your application for development or production.
+* Temporary (ephemeral) environments, to which you can temporarily deploy your application during testing.
+
+The following sections further explain these different environments and how you can use them.
+
+== Static environments
+
+By default, {ProductName} deploys your application to AWS through Red Hat OpenShift Service on AWS (ROSA). This default static environment runs your application for as long as you wish. But you can host your application in a different static environment, using a bring your own cluster (BYOC) option.
+
+Currently, with the BYOC option, you can use a static environment hosted by any of the major cloud providers—AWS, Azure, GCP, etc.  
+
+== Temporary environments   
+
+You might want to deploy your application as part of a custom integration test. For example, you might want to test if endpoints react in an expected manner, or to see if components react correctly to API queries. 
+
+Normally, {ProductName} runs integration tests in the static development environment. But to deploy your application during a test, you need a new environment. The static development environment might already host the latest version of your application. 
+
+This is where temporary environments come in. You can configure {ProductName} to deploy your application to a temporary clone of the development environment. You can perform this configuration when adding an integration test. {ProductName} then clones the environment each time it performs the test. The temporary environment disappears after testing. 
```

Comment:
I'm afraid phrasal verbs are frowned upon (more advanced vocab, hidden meaning). Will "... environments can help" be a good alternative?